app/lib/actions/katello/content_view/promote.rb

```diff
@@ -4,7 +4,7 @@ module ContentView
       class Promote < Actions::EntryAction
         middleware.use Actions::Middleware::KeepCurrentUser
 
-        def plan(version, environments, is_force = false, description = nil, options = {})
+        def plan(version, environments, is_force = false, description = nil, _options = {})
```

Comment:
is `_options` the more correct way to do it?

Reply:
that's rubocop's suggestion for unused variables.  We could remove that param here, but it seemed safer to just leave it.  I'm open to removing it out right

Reply:
If it's not a lot of investigation to remove it, I'd say just remove it. Better to do it now than later in the 3.10 cycle